docs/workflows/build-pages-with-storybook.md

```diff
@@ -38,7 +38,8 @@ When you are building screens in this way, it is typical that the inputs of a co
   paths={[
     'react/simple-page-implementation.js.mdx',
     'react/simple-page-implementation.ts.mdx',
-    'vue/simple-page-implementation.vue.mdx'
+    'vue/simple-page-implementation.vue.mdx',
+    'angular/simple-page-implementation.ts.mdx'
```

Comment:
```suggestion
    'angular/simple-page-implementation.ts.mdx',
```

Reply:
Fixed with the latest commit. Thank you for checking it :)